packages/syft/src/syft/lib/python/slice.py

```diff
@@ -47,52 +63,138 @@ def id(self) -> UID:
         return self._id
 
     def __eq__(self, other: Any) -> SyPrimitiveRet:
-        res = self.value.__eq__(other)
+        """
+        Compare if self == other.
```

Comment:
MAYBE: shall we change to `google python` style docstring format?

Reply:
Yup!

Reply:
Solved by fe2c81b